beacon-chain/node/node.go

```diff
@@ -464,7 +474,29 @@ func (b *BeaconNode) startSlasherDB(cliCtx *cli.Context) error {
 }
 
 func (b *BeaconNode) startStateGen() error {
-	b.stateGen = stategen.New(b.db)
+	o := make([]stategen.StateGenOption, 0)
+	_, err := b.db.OriginCheckpointBlockRoot(b.ctx)
+	if err == nil {
```

Comment:
Easier to follow with a switch statement here

Reply:
this was obsoleted by some other changes.